devops/actions/run-tests/cts/action.yml

```diff
@@ -19,6 +21,18 @@ inputs:
 runs:
   using: "composite"
   steps:
+  - name: Checkout cts_exclude_filter folder
+    uses: actions/checkout@v4
+    with:
+      ref: ${{ inputs.cts_exclude_ref }}
+      path: tmp
```

Comment:
can we give the folder a real name?

Reply:
Is it okay? [7ddd883](https://github.com/intel/llvm/pull/17228/commits/7ddd883935153c27a44025e3627230dbb2c42525)

Reply:
@sarnex friendly ping

Reply:
sorry i get so many emails